staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -410,6 +410,13 @@ func (c *Controller) syncLoadBalancerIfNeeded(ctx context.Context, service *v1.S
 		c.eventRecorder.Event(service, v1.EventTypeNormal, "EnsuredLoadBalancer", "Ensured load balancer")
 	}
 
+	// Don't patch the status if the Service is being deleted.
+	// Per example, if the Service isgarbage collected because the namespace is deleted,
+	// the patch will return an error and will keep retrying.
+	if service.DeletionTimestamp != nil {
```

Comment:
why does patching status to set a condition return an error just because the deletion time stamp is set?

Reply:
I may be looking into the wrong place, is not this patch operation the one I was chasing in the logs

Reply:
this is the one 
![image](https://user-images.githubusercontent.com/6450081/227066094-e147da52-68b1-4492-b71d-fd5f07250957.png)

Reply:
The event creation is what is being rejected, right?

Reply:
I need some sleep 😄